.github/actions/build-windows-image/action.yml

```
@@ -0,0 +1,38 @@
+name: "Build Windows images"
+description: "Builds wcow images with MSVC and CUDA"
+
+inputs:
+  repo:
+    type: string
+    required: true
+    description: Image repository
+  win_edition:
+    type: string
+    required: true
+    description: Windows edition
+  isolation:
+    type: string
+    required: true
+    description: Which Windows isolation mode to build for
+  vs_version:
+    type: string
+    required: true
+    description: MSVC version
+  cuda_version:
+    type: string
+    required: true
+    description: CUDA version
+
+runs:
+  using: composite
+  steps:
+    - name: Run action script to build images
+      id: build
+      shell: powershell
+      run: |
+        .github/actions/build-windows-image/action.ps1 `
+          -cudaVersion ${{ inputs.cuda_version }}      `
+          -msvcVersion ${{ inputs.vs_version }}        `
+          -edition ${{ inputs.win_edition }}           `
+          -isolation ${{ inputs.isolation }}           `
+          -repo ${{ inputs.repo }}
```

Not sure about powershell syntax, but something like this?
```suggestion
          -repo ${{ inputs.repo }}
          -prefix "$(git describe --abbrev=0 --tags | sed 's/[a-zA-Z]//g' | cut -d '.' -f -2)-"
```